client/src/components/FrequentImages.jsx

```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+
+import ItemsCart from './ItemsCart.jsx';
+import { fileURLToPath } from 'url';
+
+const FrequentImages = (props) => {
+    let mappedImages = [];
+    let filteredImages = [];
+    if (props.frequentItems !== undefined && props.frequentItems.length > 0) {
+
+        filteredImages = props.frequentItems.filter(el => {
+            return el.checked === true;
+        });
+
+        mappedImages = filteredImages.map((element, index) => {
+            if (filteredImages.length > 1 && index !== filteredImages.length - 1) {
+                return <div className="item-plus"><img src={element.image} className="item-image" key={index}/><div>+</div></div> 
```

Comment:
Similar to my previous comment above, breaking **line 17** into a few lines (as how you would break up HTML code into several lines if a new element will be rendered) would be more helpful to understand what HTML will be rendered.

Reply:
I broke up this line and several others components, especially around conditional rendering.